automation_tools/__init__.py

```diff
@@ -1910,6 +1941,7 @@ def product_install(distribution, create_vm=False, certificate_url=None,
             sat_version=satellite_version,
             host=host
         )
+    execute(setup_code_coverage, host=host)
```

Comment:
There is no need to restart the services at this stage as with all Tiers , it's all instance based, so anyways it would get started with a new instance for each Tier.